frontend/src/slices/auth/actions.ts

```diff
@@ -39,12 +42,12 @@ const getAuthenticatedUser = createAsyncThunk<
   UserAuthResponseDto | null,
   undefined,
   AsyncThunkConfig
->(`${sliceName}/get-authenticated-user`, async (_, { extra }) => {
+>(`${sliceName}/get-authenticated-user`, async (_, { extra, dispatch }) => {
   const { authApi } = extra;
-
   const hasToken = await storage.has(StorageKey.TOKEN);
-
   if (hasToken) {
+    dispatch(appActions.navigate(AppRoute.ROOT));
```

Comment:
What if the user directly opens another route, for example /meditation? I think we shouldn't redirect them to the root in that case

Reply:
yes, let's remove this

Reply:
instead let's return Navigate in auth component if user is present